mapproxy/run.sh

```diff
@@ -3,6 +3,6 @@ docker kill mapproxy
 docker rm mapproxy
 mkdir mapproxy
 chmod a+wX mapproxy
-docker run --name="mapproxy" -v `pwd`/mapproxy:/mapproxy -p 8080:8080 -d -t thinkwhere/mapproxy
+docker run --name="mapproxy" -v `pwd`home/projects/mapproxy/mapproxy-config:/mapproxy -p 8080:8080 -d -t thinkwhere/mapproxy
```

Comment:
Lines 3-5 create a folder 'mapproxy' to hold config & cache, but this line is using home/projects/mapproxy/mapproxy-config instead.  Is this just for testing?